defaults/main.yml

```diff
@@ -460,6 +461,23 @@ gitlab_shell_ssh_port: '22'
 gitlab_support_filesystem_acl: True
                                                                    # ]]]
                                                                    # ]]]
+# Custom APT package repositories [[[
+
+# -----------------------------------
+# .. envvar:: gitlab__yarn_apt_key_id [[[
+#
+# The OpenPGP key used to sign Yarn repository.
+gitlab__yarn_apt_key_id: '72ECF46A56B4AD39C907BBB71646B01B86E50310'
+
+                                                                   # ]]]
+# .. env/var:: gitlab__yarn_apt_repo [[[
+#
+# APT repository URL to Yarn repository.
+gitlab__yarn_apt_repo: 'deb https://dl.yarnpkg.com/debian/ {{ ansible_distribution_release }} main'
```

Comment:
The string needs to use `stable` and not a specific OS release to work correctly.